### ISSUE TITLE
Add SC segmentations

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/data-management#207]
From: https://github.com/ivadomed/model_seg_ms_mp2rage/blob/main/preprocessing/preprocess_data.sh
Commit: https://github.com/ivadomed/model_seg_ms_mp2rage/commit/ceb651661779896fda8a2b60dd735b0dbdb7ffd7

Name of label:
````
derivatives/labels/sub-xxx/anat/sub-xxx_UNIT1_label-SC_seg.nii.gz
````

Copy from rosenberg to local git-annex:
```bash
TYPE=C
for i in $(seq -f "%03g" 001 150); do cp ~/temp/rosenberg_nvme/data_seg_mp2rage_20230124_132415/data_processed/sub-${TYPE}${i}/anat/sub-${TYPE}${i}_UNIT1_seg.nii.gz derivatives/labels/sub-${TYPE}${i}/anat/sub-${TYPE}${i}_UNIT1_label-SC_seg.nii.gz; done
```

**Comment by jcohenadad** [comment]
Checking if there is a difference between:
~~~
derivatives/labels/sub-xxx/anat/sub-xxx_UNIT1_label-SC_seg.nii.gz
~~~
and the previously existing:
~~~
derivatives/labels/sub-Pxxx/anat/sub-Pxxx_UNIT1_seg-manual.nii.gz
~~~

<details><summary>Terminal output</summary>

```console
for i in $(seq -f "%03g" 001 250); do diff derivatives/labels/sub-${TYPE}${i}/anat/sub-${TYPE}${i}_UNIT1_seg-manual.nii.gz derivatives/labels/sub-${TYPE}${i}/anat/sub-${TYPE}${i}_UNIT1_label-SC_seg.nii.gz; done 
diff: derivatives/labels/sub-P001/anat/sub-P001_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P003/anat/sub-P003_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P008/anat/sub-P008_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P008/anat/sub-P008_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P009/anat/sub-P009_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P009/anat/sub-P009_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P010/anat/sub-P010_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P012/anat/sub-P012_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P015/anat/sub-P015_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P016/anat/sub-P016_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P018/anat/sub-P018_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P018/anat/sub-P018_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P019/anat/sub-P019_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P020/anat/sub-P020_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P020/anat/sub-P020_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P022/anat/sub-P022_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P025/anat/sub-P025_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P026/anat/sub-P026_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P029/anat/sub-P029_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P030/anat/sub-P030_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P031/anat/sub-P031_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P032/anat/sub-P032_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P032/anat/sub-P032_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P060/anat/sub-P060_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P060/anat/sub-P060_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P064/anat/sub-P064_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P064/anat/sub-P064_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P090/anat/sub-P090_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P090/anat/sub-P090_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P093/anat/sub-P093_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P093/anat/sub-P093_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P107/anat/sub-P107_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P107/anat/sub-P107_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P127/anat/sub-P127_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P127/anat/sub-P127_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P128/anat/sub-P128_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P128/anat/sub-P128_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P131/anat/sub-P131_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P131/anat/sub-P131_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P143/anat/sub-P143_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P143/anat/sub-P143_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P150/anat/sub-P150_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P150/anat/sub-P150_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P152/anat/sub-P152_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P152/anat/sub-P152_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P154/anat/sub-P154_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P154/anat/sub-P154_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P155/anat/sub-P155_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P155/anat/sub-P155_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P158/anat/sub-P158_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P158/anat/sub-P158_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P164/anat/sub-P164_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P164/anat/sub-P164_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P166/anat/sub-P166_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P166/anat/sub-P166_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P168/anat/sub-P168_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P168/anat/sub-P168_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P171/anat/sub-P171_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P171/anat/sub-P171_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P177/anat/sub-P177_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P177/anat/sub-P177_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P186/anat/sub-P186_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P186/anat/sub-P186_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P195/anat/sub-P195_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P195/anat/sub-P195_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P196/anat/sub-P196_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P196/anat/sub-P196_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P201/anat/sub-P201_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P201/anat/sub-P201_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P202/anat/sub-P202_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P202/anat/sub-P202_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P203/anat/sub-P203_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P203/anat/sub-P203_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P204/anat/sub-P204_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P204/anat/sub-P204_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P205/anat/sub-P205_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P205/anat/sub-P205_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P206/anat/sub-P206_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P206/anat/sub-P206_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P207/anat/sub-P207_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P207/anat/sub-P207_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P208/anat/sub-P208_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P208/anat/sub-P208_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P209/anat/sub-P209_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P209/anat/sub-P209_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P210/anat/sub-P210_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P210/anat/sub-P210_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P211/anat/sub-P211_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P211/anat/sub-P211_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P212/anat/sub-P212_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P212/anat/sub-P212_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P213/anat/sub-P213_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P213/anat/sub-P213_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P214/anat/sub-P214_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P214/anat/sub-P214_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P215/anat/sub-P215_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P215/anat/sub-P215_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P216/anat/sub-P216_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P216/anat/sub-P216_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P217/anat/sub-P217_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P217/anat/sub-P217_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P218/anat/sub-P218_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P218/anat/sub-P218_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P219/anat/sub-P219_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P219/anat/sub-P219_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P220/anat/sub-P220_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P220/anat/sub-P220_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P221/anat/sub-P221_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P221/anat/sub-P221_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P222/anat/sub-P222_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P222/anat/sub-P222_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P223/anat/sub-P223_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P223/anat/sub-P223_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P224/anat/sub-P224_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P224/anat/sub-P224_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P225/anat/sub-P225_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P225/anat/sub-P225_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P226/anat/sub-P226_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P226/anat/sub-P226_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P227/anat/sub-P227_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P227/anat/sub-P227_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P228/anat/sub-P228_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P228/anat/sub-P228_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P229/anat/sub-P229_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P229/anat/sub-P229_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P230/anat/sub-P230_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P230/anat/sub-P230_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P231/anat/sub-P231_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P231/anat/sub-P231_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P232/anat/sub-P232_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P232/anat/sub-P232_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P233/anat/sub-P233_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P233/anat/sub-P233_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P234/anat/sub-P234_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P234/anat/sub-P234_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P235/anat/sub-P235_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P235/anat/sub-P235_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P236/anat/sub-P236_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P236/anat/sub-P236_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P237/anat/sub-P237_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P237/anat/sub-P237_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P238/anat/sub-P238_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P238/anat/sub-P238_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P239/anat/sub-P239_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P239/anat/sub-P239_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P240/anat/sub-P240_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P240/anat/sub-P240_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P245/anat/sub-P245_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P245/anat/sub-P245_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P247/anat/sub-P247_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P247/anat/sub-P247_UNIT1_label-SC_seg.nii.gz: No such file or directory
diff: derivatives/labels/sub-P248/anat/sub-P248_UNIT1_seg-manual.nii.gz: No such file or directory
diff: derivatives/labels/sub-P248/anat/sub-P248_UNIT1_label-SC_seg.nii.gz: No such file or directory
```

</details>

all good 👍  I will then delete the "seg-manual" files:
~~~
find . -name "*seg-manual.*" -delete
~~~

**Comment by jcohenadad** [comment]
Fixed in commit: ffe427d4d1f62832e5f3567c8ce814eeff9b9764
